applications/audio/broadcast_config_tool/src/main.c

```diff
@@ -807,6 +974,27 @@ static int cmd_show(const struct shell *shell, size_t argc, char **argv)
 		shell_print(shell, "\tStreaming: %s", (streaming ? "true" : "false"));
 
 		broadcast_config_print(shell, &broadcast_param[i]);
+
+		shell_print(shell, "\t\tFiles:");
+
+		for (size_t j = 0; j < broadcast_param[i].num_subgroups; j++) {
+			for (size_t k = 0; k < broadcast_param[i].subgroups[j].num_bises; k++) {
+
+				uint8_t streamer_idx = lc3_streamer_idx[i][j][k];
+
+				if (streamer_idx == LC3_STREAMER_INDEX_UNUSED) {
+					shell_print(shell, "\t\t\tBIS %d: Not set", k);
+				} else {
+					char file_name[30];
+					bool looping = lc3_streamer_is_looping(streamer_idx);
+
+					lc3_streamer_file_path_get(streamer_idx, file_name,
+								   sizeof(file_name));
+					shell_print(shell, "\t\t\tBIS %d: %s %s", j, file_name,
+						    looping ? "(looping)" : "");
+				}
+			}
+		}
```

Comment:
Why not put this in broadcast_config_print where there is already a for-loop?

Reply:
Updated `broadcast_config_print` to take in group index so file can be printed there